Dockerfile

```diff
@@ -1,17 +1,25 @@
 FROM alpine:latest as build
 
-ADD . /sslh
-
 RUN \
   apk add \
     gcc \
     libconfig-dev \
     make \
     musl-dev \
     pcre2-dev \
-    perl && \
-  cd /sslh && \
-  make sslh-select && \
+    perl \
+    perl-dev \
+    perl-app-cpanminus \
+    git
+
+WORKDIR /sslh
```

Comment:
from a caching layer improvement point, I'd do the WORKDIR before RUN, as this is not likely to ever change.